cvmfs/catalog_mgr_rw.h

```diff
@@ -111,6 +123,8 @@ class WritableCatalogManager : public SimpleCatalogManager {
 
   manifest::Manifest *Commit(const bool     stop_for_tweaks,
                              const uint64_t manual_revision);
+  void Balance();
+  void FixWeight(WritableCatalog *catalog);
```

Comment:
It could, but I thought it could be useful to be able to individually balance a node (catalog). If this is not the case it can be perfectly private